src/handlers/generate-erc721-permit.ts

```diff
@@ -67,7 +67,8 @@ export async function generateErc721PermitSignature(
     _repositoryName = contextOrPermitPayload.repositoryName;
     _userId = contextOrPermitPayload.userId;
   } else {
-    const { NFT_MINTER_PRIVATE_KEY, NFT_CONTRACT_ADDRESS } = contextOrPermitPayload.env;
+    const { NFT_MINTER_PRIVATE_KEY = "", NFT_CONTRACT_ADDRESS = "" } = contextOrPermitPayload.env;
```

Comment:
This should throw if these are not defined rather than assigning them an empty string. Otherwise I think things are okay